test/e2e/data/text_embedding_input_v2.json

```diff
@@ -0,0 +1,10 @@
+{
+  "inputs": [
+    {
+      "name": "input-0",
+      "datatype": "BYTES",
+      "shape": [],
+      "data": ["This is an example sentence."]
+    }
+  ]
+}
```

Comment:
should we consider implementing using the openai embedding schema?
https://platform.openai.com/docs/guides/embeddings/what-are-embeddings

Reply:
We have already generated the type in kserve
https://github.com/kserve/kserve/blob/master/python/kserve/kserve/protocol/rest/openai/types/openapi.py#L769